ragstack-e2e-tests/e2e_tests/langchain/test_compatibility_rag.py

```diff
@@ -52,6 +54,16 @@ def cassandra():
 
 @pytest.fixture
 def openai_llm():
+    return ChatOpenAI(
+        openai_api_key=get_required_env("OPEN_AI_KEY"),
+        model="gpt-3.5-turbo-16k",
+        streaming=False,
+        temperature=0,
+    )
```

Comment:
Note -- this also adds the non-streaming version of OpenAI to the test matrix, as recommended.